### ISSUE TITLE
[kat::FileSystemError*] = Could not find jellyfish executable

### ISSUE THREAD

**Issue by tseemann** [TGAC/KAT#36]
I think it should be looking for `jellyfishk` ?

```
% kat hist
Kmer Analysis Toolkit (KAT) V2.0.8

inc/kat_fs.hpp(129): Throw in function kat::KatFS::KatFS(const char*)
Dynamic exception type: boost::exception_detail::clone_impl<kat::FileSystemException>
std::exception::what: std::exception
[kat::FileSystemError*] = Could not find jellyfish executable at: /bio/linuxbrew/Cellar/kat/2.0.8/bin/jellyfish
```

Maybe this code in `jellyfish_helper.cc` ?
```
path kat::JellyfishHelper::jellyfishExe = "jellyfish";
```

But this too?

```   
 string jellyfishCmd =   kat::JellyfishHelper::jellyfishExe.empty() ?
                                "jellyfish" :
                                kat::JellyfishHelper::jellyfishExe.string();
```

**Comment by maplesond** [comment]
I see.  That would explain the side issue discussed in https://github.com/TGAC/KAT/issues/31.

I think I wasn't noticing this problem because I have a jellyfish installation on my development machine.  I'll make a fix for V2.0.9.  (This is already fixed in the upcoming V2.1).

**Comment by maplesond** [comment]
Should be fixed in the latest release: https://github.com/TGAC/KAT/releases/tag/Release-2.1.0.  Please reopen if there are any issues.
